elements/lisk-transactions/test/base_transaction.spec.ts

```diff
@@ -584,8 +584,8 @@ describe('Base transaction class', () => {
 			expect((errors as ReadonlyArray<TransactionError>)[0]).toBeInstanceOf(
 				TransactionError,
 			);
-			expect((errors as ReadonlyArray<TransactionError>)[0].message).toEqual(
-				`Account does not have enough LSK: ${defaultSenderAccount.address}, balance: 0`,
+			expect((errors as ReadonlyArray<TransactionError>)[0].message).toContain(
```

Comment:
Maybe better to do exact match?

Reply:
Checking the exact match of error message make the test too fragile IMO